File: api/PizzaWorkflow/Models/Order.cs
using Newtonsoft.Json;

namespace PizzaWorkflow.Models
{
    public class Order
    {
        [JsonProperty("id")]
        public string Id { get; set; }
        public string CustomerName { get; set; }
        [JsonProperty("customerAddress")]
        public string CustomerAddress { get; set; }
        [JsonProperty("menuItems")]
        public MenuItem[] MenuItems { get; set; }
    }
}

File: src/types/Order.ts
export type Order = {
  id: string;
  customerName: string;
  customerAddress: string;
  menuItems: MenuItem[];
};

export type MenuItem = {
  type: MenuItemType;
  name: string;
};

export enum MenuItemType {
  Pizza = "Pizza",
  Drink = "Drink",
}
